scripts/linters/pylint_extensions.py

```diff
@@ -1917,6 +1917,77 @@ def process_tokens(self, tokens):
                         'single-line-pragma', line=line_num)
 
 
+class SingleSpaceAfterIfElifWhileChecker(checkers.BaseChecker):
```

Comment:
Optional: Can you please also include `SingleSpaceAfterYieldChecker` in this and remove that checker?